### DISCUSSION TITLE
Sisow recurring payments

### DISCUSSION THREAD

**Discussion by remcotolsma** [pronamic/wp-pronamic-pay#126] [Ideas]
[thysgc](https://wordpress.org/support/users/thysgc/) on **October 10, 2020 at 1:51 pm**:

> Do you support the Woocommerce Subscriptions plug-in?

**Source:** https://wordpress.org/support/topic/woocommerce-subscriptions-plug-in/

[sisow](https://wordpress.org/support/users/sisow/) on **October 12, 2020 at 1:54 pm**:

> Hello 2minacdev,
> Unfortunately, we currently do not support recurring payments. We suspect recurring payments will be available at the end of this year/ the start of next year. If there are uncertainties, please let us know.
> Kind regards,
> Sisow Technical Support

**Source:** https://wordpress.org/support/topic/woocommerce-subscriptions-plug-in/#post-13523189

We replied to Sisow on **March 30, 2021**:

> @sisow Are recurring payments now possible at Sisow? I could not find any information about this at https://www.sisow.nl/. Can you give an update @sisow? Thanks.

<img width="696" alt="Schermafbeelding 2021-03-30 om 13 41 51" src="https://user-images.githubusercontent.com/869674/121154088-ef1f3d80-c846-11eb-9d82-df89e3966537.png">

Unfortunately this comment has been removed and the topic closed to new replies. Maybe it has something to do with the takeover of Sisow by Buckaroo:

- https://www.buckaroo.nl/blog/buckaroo-neemt-betaalaanbieder-sisow-over
- https://www.sisow.nl/nieuws/buckaroo-heeft-overname-sisow-afgerond/

Currently there are no indications that Recurring Payments support is available from Sisow.

**Comment by rvdsteege** [comment]
Closing this discussion as Sisow doesn't exist anymore.
